src/txHashBuilder.c

```diff
@@ -1183,25 +1187,31 @@ static void txHashBuilder_assertCanLeaveMint(tx_hash_builder_t* builder)
 	ASSERT(builder->multiassetData.remainingTokens == 0);
```

Comment:
I'd say that these two asserts should only be done for `case TX_HASH_BUILDER_IN_MINT:`?
If I am right, the change should be made in multisig already.

Reply:
I assume the data is zeroed out anyways, so if no multiassets were used (in mint or output), they should remain zero, unless pool certificate, in which case those two should be zero. But yes, logically this check is only needed in the mint case.

Reply:
It might be zeroed, but it might be not --- it's wrong to make assumptions about data that are in some union possibly used for a different purpose at the moment.

Reply:
I agree, done.